src/library/scala/collection/IterableOnce.scala

```diff
@@ -732,7 +748,12 @@ trait IterableOnceOps[+A, +CC[_], +C] extends Any { this: IterableOnce[A] =>
     *           `op( op( ... op(x,,1,,, x,,2,,) ..., x,,n-1,,), x,,n,,)` where `x,,1,,, ..., x,,n,,`
     *           are the elements of this $coll.
     *  @throws UnsupportedOperationException if this $coll is empty.   */
-  def reduceLeft[B >: A](op: (B, A) => B): B = {
+  def reduceLeft[B >: A](op: (B, A) => B): B = this match {
+    case seq: IndexedSeq[A @unchecked] if seq.length > 0 =>
+      foldl(seq, 1, seq(0), op)
+
+    case _ =>
+
```

Comment:
indentation is off below

Reply:
I can't see it in the github UI. Is this scala 2 or 3? I'll have to look at my local file. Modern github tech is so amazing. I'm usually impeccable on indents, unless it is happy hour.

Reply:
oh I resolved the comment so I could see the alignment. I think that was intentional because Scala 3 has messed with my sense of indent. Also I prefer to avoid the diff if possible.

For example, should the `case` and `if` guard align?

I don't mind indenting before squashing.